src/VirtualClient/VirtualClient.Actions/MLPerfTraining/MLPerfTrainingExecutor.cs

```diff
@@ -0,0 +1,515 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace VirtualClient.Actions
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.IO.Abstractions;
+    using System.Linq;
+    using System.Runtime.InteropServices;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.DependencyInjection;
+    using VirtualClient.Common;
+    using VirtualClient.Common.Extensions;
+    using VirtualClient.Common.Platform;
+    using VirtualClient.Common.Telemetry;
+    using VirtualClient.Contracts;
+
+    /// <summary>
+    /// The MLPerf Training workload executor.
+    /// </summary>
+    [UnixCompatible]
+    public class MLPerfTrainingExecutor : VirtualClientComponent
```

Comment:
I would still like to see this merged with MLPerfExecutor if that's doable

Reply:
As Prachi has mentioned [above](https://github.com/microsoft/VirtualClient/pull/119#discussion_r1252258000), we could merge them once we have the workloads working over A100 as well as H100 GPUs.